tests/tests.h

```diff
@@ -32,6 +32,11 @@
 #include <sstream>
 #include <iomanip>
 
+// silence extra diagnostics in the testsuite
+#ifdef DEAL_II_DISABLE_EXTRA_DIAGNOSTICS
+DEAL_II_DISABLE_EXTRA_DIAGNOSTICS
+#endif
```

Comment:
Because we hit 3736 warnings within the testsuite resulting in 3MB of additional output that has to be stored and processed by the cdash instance.

Please also note that ```DEAL_II_DISABLE_EXTRA_DIAGNOSTICS``` does <i>not</i> disable <i>all</i> diagnostics, but only - in essence - ```-Wextra``` (the category, not the compiler flag), ```-Wunused-*```, and two more.